### ISSUE TITLE
Addon Autocompletion for virtual keyboard lets kodi hang

### ISSUE THREAD

**Issue by peno64** [xbmc/xbmc#19420]
### Describe the bug
The addon Autocompletion for virtual keyboard in the kodi repository lets kodi hang when a search string  is entered and then a proposed hit is chosen

The addon Autocompletion for  virtual keyboard extends the kodi search function in a way that it searches in youtube,  google, bing, ... for best matches with what was entered in the search and then from the suggestions one can choose one with  the mouse. When this is done whole kodi hangs.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

What you have chosen with the mouse should be entered automatically in the search input and the user can then continue with  it.

## Actual Behavior

Kodi just hangs.

### To Reproduce

1.  Do a new fresh install of kodi 19, I did it under windows
2. Install the youtube addon from the kodi repository
3. Install the Autocompletion for virtual keyboards addon from the kodi repository
4. Start youtube. No need to define a api key
5. Choose Search, New search
6. Enter start trek
7. Suggestions are shown below the search
8. Choose for example Start Trek trailer
9. Kodi hangs and must be killed

### Debuglog

The debuglog can be found here:
https://paste.kodi.tv/beyukelume.kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: 19.0.0

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by sualfred** [comment]
@peno64 
I've tested it with GlobalSearch and some other addons without any issues. Just the YT addon hangs in general on my end. With the new "Tubed" addon, it works as it should.

**Comment by peno64** [comment]
Yes with global search it doesn't hang. Please try with the original youtube addon, not the new Tubed addon.

**Comment by peno64** [comment]
But even with global search it doesn't work as it should because  when you clock start trek trailer, this isn't then entered in the search. The entered texts star trek stays there while it should be replaced by start trek trailer

**Comment by sualfred** [comment]
Since the old YT addon isn't working on my end at all -> two options:
a) It's an issue of the old YT addon (use Tubed instead. It's brandnew from scratch)
b) It's an Kodi internal "busydialog" bug.

I can confirm that the first suggestion isn't filled correctly, all others are. WIll take a look at it as soon as I find some time.

**Comment by sualfred** [comment]
Just looked at the log, please provide a debug log.

**Comment by peno64** [comment]
youtube is not the only addon where it hangs.
For example install the addon vrt nu from the kodi repo, then start it and choose "Search VRT NU" Also enter start trek and then choose start trek trailer. Kodi hangs.
This is the debug log with this: https://paste.kodi.tv/osovujejir.kodi

There are many other addons where it hangs

**Comment by peno64** [comment]
Btw, it also hangs with tubed. This is the log with it: https://paste.kodi.tv/icayevubak.kodi

**Comment by sualfred** [comment]
I wish I could tell u something different, but it doesn't hang on my end. It's super instant.

**Comment by peno64** [comment]
Very strange. Did you also do a clean install on windows? Also note that I start kodi portable. So I did a setup op kodi windows x64 and then in a command prompt I start kodi with the command kodi -p
It allows you to have multiple kodi's on one system without interfering each other. But I don't think it has to do with portable because I know someone who has it also on android, normal install.

**Comment by sualfred** [comment]
U don't have to tell me how to start Kodi in portable mode. I'm using different dev installations on my PC for testings. 

I'm asking other team members if somebody is able to reproduce it, but since your "fix" is just clearing a busydialog window, I beg it's Kodi core related.
